adapters/Abstract.cfc

```diff
@@ -165,7 +165,7 @@
 		// define the foreign keys
 		loc.iEnd = ArrayLen(arguments.foreignKeys);
 		for (loc.i=1; loc.i <= loc.iEnd; loc.i++) {
-			loc.sql = loc.sql & ",#chr(13)##chr(10)# " & arguments.foreignKeys[loc.i].toSQL();
+			loc.sql = loc.sql & ",#chr(13)##chr(10)# " & arguments.foreignKeys[loc.i].toForeignKeySQL();
```

Comment:
I left the original .toSQL() function unchanged and simply reference my new function instead